### ISSUE TITLE
[Bug]: @php blocks handled differently when nested in @if blocks

### ISSUE THREAD

**Issue by garrettboone** [shufo/blade-formatter#914]
### Description

Please reference https://github.com/shufo/blade-formatter/pull/909#issuecomment-1962253902

Here I show a screenshot where the `@php` block is severely wrapped but another block below is unaffected. I had only noticed the changed after upgrading from 1.32 and believe it is a difference due to the bugfix in 1.34.1. 

### Expected Behavior

But I am confused now...I thought the line length might already ignore the indentation level. Apparently not. I guess it was not getting wrapped, though I don't know why the constant of printWidth wasn't working when I had 120 set for wrap length anyway.

### Actual Behavior

Is the length of the wrap always including the indentation space? Is there a way to exempt indentation?

Is the difference in `@php` block formatting expected when nested in an `@if` block? The above section wraps after an `=` but the lower section does not.

### Additional Context

v1.41.0

**Comment by claytonrcarter** [comment]
@garrettboone I can't reproduce what you're showing in #909, even when using an 80 char width. Can you share more detail? Specifically, can you share the before/after formatting, as well as what you expect it to look like? Something that can be copy/pasted, please.

I am assuming that the first block is more severely wrapped just because it contains more code. The second block is very short and makes sense that it would be unaffected. Also, I recall that single statement PHP blocks (eg the second one) may be handled differently from multistatement blocks (eg the first).

**Comment by garrettboone** [comment]
With 1.41.0, and with the following config in `.bladeformatterrc.json` I am getting these results.

I've taken the `@php` out of nesting  to focus solely on line length.
Config:
```
{
  "indentSize": 4,
  "wrapAttributes": "auto",
  "wrapLineLength": 120,
  "endWithNewLine": true,
  "endOfLine": "LF",
  "useTabs": false,
  "sortTailwindcssClasses": true,
  "sortHtmlAttributes": "none",
  "noMultipleEmptyLines": true,
  "noPhpSyntaxCheck": false
}
```

I am showing the longest line to be 101 characters.

Before:
```blade
<div>
    <div>
        <div>
            <div>
                <div>
                    <div>
                        <!-- Page Content -->
                        <div id="content" class="patient-view-content">
                            <nav class="navbar navbar-expand-lg navbar-light">
                                <div class="container-fluid">
                                    <button type="button" id="sidebarCollapse" class="btn">
                                        <i class="fas fa-align-left"></i>
                                    </button>
                                </div>
                            </nav>
                            @php
                                $severe_array = false;
                                if ($patient->patient_allergies) {
                                    $severe_array ='';
                                    foreach (json_decode($patient->patient_allergies) as $allergy) {
                                        if ($allergy->severity === 'Severe') {
                                            $severe_array .= $allergy->description . ',';
                                        }
                                    }
                                }
                            @endphp
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
```

After:
```blade
<div>
    <div>
        <div>
            <div>
                <div>
                    <div>
                        <!-- Page Content -->
                        <div id="content" class="patient-view-content">
                            <nav class="navbar navbar-expand-lg navbar-light">
                                <div class="container-fluid">
                                    <button type="button" id="sidebarCollapse" class="btn">
                                        <i class="fas fa-align-left"></i>
                                    </button>
                                </div>
                            </nav>
                            @php
                                $severe_array = false;
                                if (
                                    $patient->patient_allergies
                                ) {
                                    $severe_array =
                                        '';
                                    foreach (
                                        json_decode(
                                            $patient->patient_allergies,
                                        )
                                        as $allergy
                                    ) {
                                        if (
                                            $allergy->severity ===
                                            'Severe'
                                        ) {
                                            $severe_array .=
                                                $allergy->description .
                                                ',';
                                        }
                                    }
                                }
                            @endphp
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
```

So what causes this?

**Comment by claytonrcarter** [comment]
OK, yep, I can reproduce this now; thank you. This will also be fixed by #916.
